packages/material-ui-codemod/README.md

```diff
@@ -125,11 +198,24 @@ Updates the Box API from separate system props to `sx`.
 <!-- #default-branch-switch -->
 
 ```sh
-npx @material-ui/codemod@next v5.0.0/box-sx-prop <path>
+npx @material-ui/codemod@next v5.0.0/box-borderradius-values <path>
 ```
 
 You can find more details about this breaking change in [the migration guide](https://next.material-ui.com/guides/migration-v4/#box).
 
+#### `box-rename-css`
+
+Renames the Box `css` prop to `sx`
+
+```diff
+-<Box css={{ m: 2 }}>
++<Box sx={{ m: 2 }}>
+```
+
+```sh
+npx @material-ui/codemod@next v5.0.0/box-rename-css <path>
+```
+
```

Comment:
Up until now, we have linked the codemod with the migration guide (both ways). Should we do the same with the new codemods? This:

```suggestion

You can find more details about this breaking change in [the migration guide](https://next.material-ui.com/guides/migration-v4/#system).

```

and update the migration guide too.

Reply:
sure, I will try to link to migration docs